app/utils/ui/ui.ts

```diff
@@ -1,3 +1,4 @@
+import { Appearance } from "react-native-appearance";
```

Comment:
Could you use the react native solution instead of the npm package? https://reactnative.dev/docs/appearance

Reply:
sure